### DISCUSSION TITLE
"Select" Type Cell

### DISCUSSION THREAD

**Discussion by niceiq** [andmarti1424/sc-im#841] [Ideas]
Hi, I'd like to know if there are plans to implement "select" type cells. 

Basically the ones with a drop-down menu with a few choices and you can scroll through them to chose (or being typing and there is autocomplete for the available selections). This would be tremendously useful for a lot of use cases when recording data. 

I know I'm not the only one who would benefit it. I use sc-im daily and it would really save me quite a lot of key presses.

Thank you.

P.S. The attached image is what the requested feature looks like in Google Sheet.
![image](https://github.com/andmarti1424/sc-im/assets/98485752/8711dce6-d797-4565-a20b-6add3e01e0c1)

**Comment by andmarti1424** [comment]
Hello. Not really. Not in the near future at least. my plan is to make sc-im more robust before adding some extra features
